File: Java/Bit_Manipulation/_01_OddEven.java
// 01} Check a number if it's odd or even.
public class _01_OddEven {
    public static void main(String[] args) {
        int num = 29;
        if ((num & 1) == 0)
            System.out.println("Even");
        else
            System.out.println("Odd");
    }
}

File: C++/Bit_Manipulation/_01_OddEven.cpp
// 01} Check a number if it's odd or even.
#include <iostream>
using namespace std;
int main()
{
    int num;
    cin>>num;
    if ((num & 1) == 0)
        cout<<"Even"<<endl;
    else
        cout<<"Odd"<<endl;
}